hazelcast-code-generator/src/main/resources/codec-template-md.ftl

```diff
@@ -333,3 +335,16 @@ Header only event message, no message body exist.
     </#switch>
 </#function>
 
+
+<#function resolvePartitionIdentifier partitionIdentifier>
+    <#switch partitionIdentifier?trim>
+        <#case "random">
+            <#return "a random partition id from 0 to PARTITION_COUNT(inclusive)">
```

Comment:
Can we write somewhere 'n the document what value is PARTITION_COUNT (starts from 0 or not)?